clust/utils.q

```diff
@@ -57,7 +62,7 @@ clust.i.df.nege2dist:{[data]
 // @private
 // @kind dictionary
 // @category clustUtility
-// @fileoverview Linkage dictionary
+// @desc Linkage dictionary
```

Comment:
Any non `functional` code should contain an additional `@type` param as per the coding docs 
i.e // @type dictionary